frontend/src/views/HomeView.vue

```diff
@@ -82,7 +82,7 @@ const { getConfig } = storeToRefs(useConfigStore());
         <div class="flex align-items-left">
           <p class="text-xl">
             You can assign custom permissions to other users through IDIR or BCeID authentication.<br /><br />
-            You can also choose to share files with public links.
+            IDIR users can also choose to share files with public links.
```

Comment:
It's fine now to get it out of the door, but we should eventually change this to be templated and dynamically enumerated based on the config list of elevatedRights idps so that it's more generic in design.